src/olcs/FeatureConverter.js

```diff
@@ -404,15 +410,29 @@ exports.prototype.olLineStringGeometryToCesium = function(layer, feature, olGeom
       // always update Cesium externs before adding a property
       positions,
       width,
-      vertexFormat: appearance.vertexFormat
     };
-    outlinePrimitive = new Cesium.Primitive({
+    let geometry;
+    const primitiveOptions = {
       // always update Cesium externs before adding a property
-      geometryInstances: new Cesium.GeometryInstance({
-        geometry: new Cesium.PolylineGeometry(geometryOptions)
-      }),
       appearance
-    });
+    };
+    if (heightReference == Cesium.HeightReference.CLAMP_TO_GROUND) {
+      geometry = new Cesium.GroundPolylineGeometry(geometryOptions);
```

Comment:
Can you remove the `let geometry` above and change to `const geometry = ...` here and in the else below?